doc/man3/SSL_CTX_set_session_ticket_cb.pod

```diff
@@ -100,19 +100,54 @@ Same as B<SSL_TICKET_SUCCESS>, but a new ticket should be sent to the client.
 
 =back
 
+The return value can be any of these values:
+
+=over 4
+
+=item SSL_TICKET_RETURN_ERROR
+
+An error occurred.
+
+=item SSL_TICKET_RETURN_NO_TICKET
```

Comment:
I would rename this to `SSL_TICKET_RETURN_IGNORE`

Reply:
Done